src/style.css

```diff
@@ -1,3 +1,124 @@
 body {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 400;
+  --button-color: rgb(0, 172, 220);
+  --main-text-color: rgb(6, 11, 53);
+  --secondary-text-color: rgb(97, 96, 112);
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+.card {
```

Comment:
this file should be named after the block, also move all styles not related to the block from this file

Reply:
What file are you talking about? What should it be named? I don't understand